spec/factories/offsprings.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :offspring do
+    first_name "MyString"
+    last_name "MyString"
+    type ""
```

Comment:
The idea that @amaurygonzalez and I are discussing is to use it like:
`FactoryGirl.create (:GradedOffspring, first_name: "Peter", last_name: "Richards", type: "Graded")`

****similar use with AgedOffspring**